### DISCUSSION TITLE
How is the Sitepoint logo matching 100%?

### DISCUSSION THREAD

**Discussion by M4N0JKUM4R** [cssbattle/forum#46] [Bugs reports]
![image](https://user-images.githubusercontent.com/11471878/114190041-7d229900-9968-11eb-8c42-7ee8a340bc98.png)

I noticed it at last when I was trying to enter submit and though I would get 95+ because of the missing border-radius on two points of the logo.

**Comment by bmerigan** [comment]
Wow. lucky you.
It seems to be quite relaxed on this one - I have 100% too and can see a slight jitter difference still.
